### PR TITLE
[MON-143969] - Documentation_improve discoverability of help request buttons

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: src/theme/TOC/index.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+import { useDoc } from '@docusaurus/theme-common/internal';
+import styles from './styles.module.css';
+import Translate from '@docusaurus/Translate';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+export default function TOC(props) {
+  const [showFooterButtons, setShowFooterButtons] = useState(false);
+  const location = useLocation();
+  const { metadata } = useDoc();
+  const imagePath = useBaseUrl('/img/image.png');
```
Comment (kduret) [inline src/theme/TOC/index.js:14]:
```suggestion
  const imagePath = './img/image.png';
```

File: src/theme/TOC/index.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+import { useDoc } from '@docusaurus/theme-common/internal';
+import styles from './styles.module.css';
+import Translate from '@docusaurus/Translate';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+export default function TOC(props) {
+  const [showFooterButtons, setShowFooterButtons] = useState(false);
+  const location = useLocation();
+  const { metadata } = useDoc();
+  const imagePath = './img/image.png';
```
Comment (cg-tw) [inline src/theme/TOC/index.js:14]:
```suggestion
  const imagePath = useBaseUrl('/img/image.png');
```

File: src/theme/TOC/index.js [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  const handleToggleButtons = () => {
+    setShowFooterButtons(prev => !prev);
+  };
+
+  return (
+    <div>
+      <DefaultTOC {...props} />
+
+      <div className={styles.pngContainer}>
+        {showFooterButtons && (
```
Comment (Thebarda) [inline src/theme/TOC/index.js:27]:
```suggestion
```

File: src/theme/TOC/index.js [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+                <Translate
+                  id="theme.common.theWatchButton"
+                  description="The link label for the -the watch- button"
+                >
+                  Ask on The Watch
+                </Translate>
+              </span>
+            </a>
+          </div>
+        )}
```
Comment (Thebarda) [inline src/theme/TOC/index.js:152]:
```suggestion
```

File: src/theme/TOC/index.js [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+    setShowFooterButtons(prev => !prev);
+  };
+
+  return (
+    <div>
+      <DefaultTOC {...props} />
+
+      <div className={styles.pngContainer}>
+        {showFooterButtons && (
+          <div className={clsx(styles.footerButtons, { [styles.showButtons]: showFooterButtons })}>
```
Comment (Thebarda) [inline src/theme/TOC/index.js:28]:
```suggestion
          <div className={clsx(styles.footerButtons)}>
```

File: src/theme/TOC/styles.module.css [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  opacity: 0;
+  transition: opacity 0.4s ease-in-out;
+  margin-left: 10px;
+  text-align: left;
+  flex-grow: 1;
+}
+
+.footerButtons a:hover .buttonText {
+  opacity: 1;
+}
```
Comment (Thebarda) [inline src/theme/TOC/styles.module.css:77]:
```suggestion
}

.editPage, .giveFeedback, .theWatch {
  bottom: 160px;
  transition: bottom 300ms ease-out;
}

.editPageOpen {
    bottom: 220px;
}

.giveFeedbackOpen {
  bottom: 280px;
}

.theWatchOpen {
  bottom: 320px:
}
```

File: src/theme/TOC/index.js [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+            {editUrl && (
+              <a href={editUrl} className={clsx(styles.buttonFilled)}>
```
Comment (Thebarda) [inline src/theme/TOC/index.js:30]:
```suggestion
              <a href={editUrl} className={clsx(styles.editPage, [styles.editPageOpen]: showFooterButtons)}>
```

File: src/theme/TOC/index.js [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+                    Éditer cette page
+                  </Translate>
+                </span>
+              </a>
+            )}
+            <a
+              rel="noreferrer noopener"
+              href={`https://docs.google.com/forms/d/e/1FAIpQLSdeKkEHWbxn4ZweR1M4OEMzP_QoqKmjl6t4iC_yKeX2CFjzAw/viewform?usp=pp_url&entry.371845164=docs.centreon.com${location.pathname}`}
+              target="_blank"
+              className={clsx(styles.buttonFilled)}
```
Comment (Thebarda) [inline src/theme/TOC/index.js:59]:
```suggestion
              className={clsx(styles.giveFeedback, [styles.giveFeedbackOpen]: showFooterButtons)}
```

**Review by cg-tw — APPROVED** [review]

**Review by Wedierre — APPROVED** [review]

**Review by mushroomempires — APPROVED** [review]
approving to unlock merge although the PR doesn't target anything related to owners-pipelines territory

**Pull request merged**